### PR TITLE
GDB and Octave

### PR DESCRIPTION
**Pull request by amisha2016**

### CODE REVIEW

**Comment by mandeeps708** [comment]
@amarjeetkapoor1 please review this one.

**Comment by amarjeetkapoor1** [comment]
@mandeeps708 Any particular thing that you want me to review?

**Comment by mandeeps708** [comment]
No. Just the pull request as a general.
Which includes seeing commit messages, if no. of commits are necessary to do the thing etc.

**Comment by amarjeetkapoor1** [comment]
@amisha2016 Please recheck the code.

File: _posts/2016-06-29-octave.markdown [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "GNU Debugger"
```
Comment (amarjeetkapoor1) [inline _posts/2016-06-29-octave.markdown]: Its Octave

**Comment by amisha2016** [comment]
I have update the files. Now. I think its up to right. You can merge it.

**Pull request merged**